scripts/vim-patch.sh

```diff
@@ -179,6 +179,16 @@ preprocess_patch() {
   # Rename path to matchit plugin.
   LC_ALL=C sed -e 's@\( [ab]/runtime\)/pack/dist/opt/matchit/\(plugin/matchit.vim\)@\1/\2@g' \
     "$file" > "$file".tmp && mv "$file".tmp "$file"
+  LC_ALL=C sed -e 's@\( [ab]/runtime\)/pack/dist/opt/matchit/doc/\(matchit.txt\)@\1/doc/pi_\2@g' \
+    "$file" > "$file".tmp && mv "$file".tmp "$file"
+
+  # Rename test_urls.vim to check_urls.vim
+  LC_ALL=C sed -e 's@\( [ab]\)/runtime/doc/test\(_urls.vim\)@\1/scripts/check\2@g' \
+    "$file" > "$file".tmp && mv "$file".tmp "$file"
+
+  # Rename path to check_colors.vim
+  LC_ALL=C sed -e 's@\( [ab]/runtime\)/colors/\(tools/check_colors.vim\)@\1/\2@g' \
```

Comment:
Is this intended for end-users or Vim/Neovim developers? I put this script in `runtime/tools/` but `tools/` is not mentioned in `:h runtimepath`. End users would have difficulty finding this and they have to read `runtime/colors/README.txt` for more info.

There is https://github.com/lifepillar/vim-colortemplate for creating a colorscheme from a template.

Reply:
It's for developers who want to submit a new colorscheme to Vim core.

Reply:
![image](https://user-images.githubusercontent.com/53520949/161988033-427ebd1a-8a32-4e8e-a750-a6968a2dd891.png)
The file is missing

Reply:
It's present in the Neovim source tree but not installed.